src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      <p className="Person__partner">
+        {person.isMarried ? (
+          <>
+            {person.partnerName} is my {partner}
+          </>
+        ) : (
+          <>I am not married</>
+        )}
```

Comment:
```suggestion
        {person.isMarried 
          ?  `${person.partnerName} is my ${partner}`
          : 'I am not married'
        }
```
no need to wrap in fragment text. it will  be just string.
Also will be better move this ternary in component (below partner ternary) and render just final text in jsx.